model-archiver/model_archiver/model_packaging_utils.py

```diff
@@ -57,11 +57,10 @@ def check_mar_already_exists(model_name, export_file_path, overwrite):
             if overwrite:
                 logging.warning("Overwriting %s ...", export_file)
             else:
-                logging.error("%s already exists.\n"
-                              "Please specify --force/-f option to overwrite the model archive output file.\n"
-                              "See -h/--help for more details.", export_file)
-
-                sys.exit(1)
+                raise ModelArchiverError("%s already exists.\n"
+                                         "Please specify --force/-f option to overwrite the model archive "
```

Comment:
export_file_path is expected to be folder, We should check export_file_path exists and it's has to be a folder.

Reply:
This is already handled. You get "Not a directory:" error

Reply:
```
$ model-archiver --model-name alexnet --model-path onnx_alexnet/ --export-path /tmp/alexnet.mar -f --handler mxnet_vision_service:handle
ERROR:root:Failed to save the model-archive to model-path "/tmp/alexnet.mar". Check the file permissions and retry.
ERROR:root:[Errno 20] Not a directory: '/tmp/alexnet.mar/alexnet.mar'
Traceback (most recent call last):
  File "/Users/dantu/PycharmProjects/mxnet-model-server/model-archiver/model_archiver/model_packaging.py", line 40, in package_model
    ModelExportUtils.zip(export_file_path, model_name, model_path, files_to_exclude, manifest)
  File "/Users/dantu/PycharmProjects/mxnet-model-server/model-archiver/model_archiver/model_packaging_utils.py", line 232, in zip
    with zipfile.ZipFile(mar_path, 'w', zipfile.ZIP_DEFLATED) as z:
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/zipfile.py", line 1090, in __init__
    self.fp = io.open(file, filemode)
NotADirectoryError: [Errno 20] Not a directory: '/tmp/alexnet.mar/alexnet.mar'
```
I might be missing something, would like to understand what it is. Is the suggestion to make sure that we catch "not a directory" error or to modify the error string?

Reply:
We should not print stacktrace for input error cases. Instead we should give clear error message.

And the format of message should not be normally logging format. We should provide a good user experience as a normal command line tool.

Reply:
Ok. I am guessing you are asking to validate the input params. We could add that.